canonical-web-and-design/react-components
# Update Vanilla to 2.19.2, fix hiding table columns

## Done

- Updates Vanilla to 2.19.2
- Removes unnecessary custom styling from ModularTable (this is now provided by Vanilla)
- Updates expanding MainTable to use `aria-hidden` instead of `u-hide` utility

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Check if ModularTable examples work correctly
- Check if MainTable expanding renders correctly
